README.rst

```diff
@@ -40,13 +40,14 @@ It's important that you run ``coverage-badge`` from the directory where the
 
 The full usage text::
 
-    usage: __main__.py [-h] [-o FILEPATH] [-q] [-v]
+    usage: __main__.py [-h] [-o FILEPATH] [-g HTML] [-q] [-v]
```

Comment:
Well, it needs to be consistent, and the argument is defined as `-n` (see https://github.com/dbrgn/coverage-badge/pull/1/files#diff-59ad6794e1dd5e79d9c48b11b7953128R67), so the documentation should say the same :)